observing/obsstate.py

```diff
@@ -0,0 +1,189 @@
+import os
+import getpass
+import time
+from pydantic import BaseModel
+import sqlite3
+from observing import parsesdf
+import logging
+
+logger = logging.getLogger(__name__)
+
+# TODO: figure out how to make it r/w for all users
+#DBPATH = 'file:/opt/devel/pipeline/ovrolwa.db?mode=rw'
+DBPATH = '/home/pipeline/proj/lwa-shell/lwa-observing/ovrolwa.db'
+
+class Session(BaseModel):
+    PI_ID: str
+    PI_NAME: str
+    PROJECT_ID: str
+    SESSION_ID: str 
+    SESSION_MODE: str
+    SESSION_DRX_BEAM: str
+    CONFIG_FILE: str
+    CAL_DIR: str
+    STATUS: str  # e.g., "scheduled" "completed"
+
+
+class Settings(BaseModel):
+    time_loaded: str
+    user: str
+    filename: str
+    time_file: str
+
+
+class Calibrations(BaseModel):
+    time_loaded: str
+    filename: str
+    beam: str
+
+
+class Product(BaseModel):
+    time_loaded: str
+    filename: str
+    beam: str
+
+
+def create_db():
+    """Create database if it doesn't exist."""
+    conn = sqlite3.connect(DBPATH)
+    c = conn.cursor()
+    c.execute('''
+        CREATE TABLE IF NOT EXISTS sessions
+        (PI_ID text, PI_NAME text, PROJECT_ID text, SESSION_ID text, SESSION_MODE text, SESSION_DRX_BEAM text, CONFIG_FILE text, CAL_DIR text, STATUS text)
```

Comment:
Good points. I added "time_loaded" as a field for all tables (in another branch), so I am starting to use that as a primary key. It isn't a great choice, since the sessions table can be uniquely identified by SESSION_ID.
I'll postpone the question of primary key for now, but would appreciate your ideas on how to do this well in a future commit.